pkgs/libresoc/openpower-isa.nix

```diff
@@ -25,6 +25,13 @@ with python39Packages;
     prePatch = ''
       # broken upstream, required for importing modules in tests
       touch ./src/openpower/{sv,test/general}/__init__.py
+
+      # ignore $CC/$AS/etc environment variables and update hard-coded prefixes with pkgsCross compiler (powerpc64le-unknown-linux-gnu-gcc)
```

Comment:
doing this substitution because this compiler prefix needs to agree between nix and openpower